src/TinkerServiceProvider.php

```diff
@@ -26,6 +26,10 @@ public function register()
         });
 
         $this->commands(['command.tinker']);
+
+        $this->publishes([
```

Comment:
I wrapped this in a `if ($this->app->runningInConsole())` but I don't use Lumen so I'm not sure what needs to be done to ensure compatibility…